Core/Analyzers/ClassRefAnalyzer.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "../Analyzer.h"
+
+namespace ObjectiveNinja {
+
+/**
+* Analyzer for extracting Objective-C class information.
+ */
```

Comment:
Formatting is weird here. Please run `clang-format -i --style=WebKit` on all of the files touched by this PR.